src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Channel.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" ?>
+
+<resources xmlns="https://api-platform.com/schema/metadata"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="https://api-platform.com/schema/metadata https://api-platform.com/schema/metadata/metadata-2.0.xsd"
+>
+    <resource
+        class="%sylius.model.channel.class%"
+        shortName="Channel"
+    >
+        <attribute name="normalization_context">
+            <attribute name="groups">
+                <attribute>channel:read</attribute>
+            </attribute>
+        </attribute>
+
+        <collectionOperations />
+
+        <itemOperations>
+            <itemOperation name="get">
+                <attribute name="groups">
+                    <attribute>channel:read</attribute>
+                </attribute>
+            </itemOperation>
+        </itemOperations>
+
+        <property name="id" identifier="true" writable="false" readable="true" />
```

Comment:
I guess that the code should an identifier to be consistent with most others resources